### ISSUE TITLE
Upsert support

### ISSUE THREAD

**Issue by gabrielittner** [requery/sqlite-android#70]
It would great if you could add methods for upsert that was added in 3.24.0 https://sqlite.org/lang_UPSERT.html

**Comment by npurushe** [comment]
Yes have been looking into this, it's a bit tricky because it's a separate clause that does the update (ON CONFLICT(x) DO UPDATE) which can have a different set of values then the insert.
